docs/documentation/writing-style-guide.md

```diff
@@ -0,0 +1,297 @@
+# Writing Style Guide
+
+Thoth Tech
+
+# Top tips for Thoth Tech writing
+
+Effective writing is clear, easy to read, and concise. Thoth Tech writing should follow the guidelines of plain English (Australian English).
+
+Here at Thoth Tech, we aim for all our writing to meet these goals:
+
+**To Empower:** We want our people to have the knowledge to make the most of our products
+
+**To Respect:** We want our people to feel included and accepted for who they are
+
+**To Educate:** We want to assist our people in their learning and provide them with the information they need
+
+**To Guide:** We want to lead our people in a thoughtful manner towards their goals
+
+**To be Honest:** We want to be truthful to our people and focus on our real strengths
+
+While writing, it is important to:
+
+1. Know your audience
+2. Know you intention of the writing
```

Comment:
Typo :) - (Know the intention of your writing?)

Reply:
```suggestion
2. Know the intention of your writing
```